pkg/webhooks/webhooks.go

```diff
@@ -112,13 +116,13 @@ func Start(ctx context.Context, cfg *rest.Config, ctors ...knativeinjection.Cont
 			ConfigMap:      lo.Must(metrics.NewObservabilityConfigFromConfigMap(nil)).GetConfigMap().Data,
 			Secrets:        sharedmain.SecretFetcher(ctx),
 			PrometheusPort: options.FromContext(ctx).WebhookMetricsPort,
-		}, logger))
+		}, logging.NewLogger(ctx, component)))
 		// Register webhook metrics
 		webhook.RegisterMetrics()
 
 		wh, err = webhook.New(ctx, webhooks)
 		if err != nil {
-			knativelogging.FromContext(ctx).Fatalw("Failed to create webhook", zap.Error(err))
+			log.Error(err, "Failed to create webhook")
```

Comment:
Same comment here. I think we should try to keep the severities if we can and still crash like we did before